js/data.js

```diff
@@ -1,16 +1,17 @@
+import { randomNumber, getRandomFloat, getRandomArrayElement, generateRandomElementsArray } from './util.js';
+
```

Comment:
В этом нет ошибки, но лучше отформатировать. Так импортируемые элементы более читаемыми и легко доступными для обозрения. Такой формат особенно удобен, когда количество импортируемых элементов увеличивается.

```suggestion
import { 
   randomNumber, 
   getRandomFloat, 
   getRandomArrayElement, 
   generateRandomElementsArray 
} from './util.js';

```